### PR TITLE
:bookmark: release 1.16.0

### PR DESCRIPTION
**Pull request by annashamray**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+* [#1611] Add importing authorization data via yaml files as a configuration step
 * [#1817] Updated OAF version to 0.9.0. This upgrade allows admin users managing their sessions through the admin.
 
+**Bugfixes and QOL**
+
+* [#1741] Clean orphaned ``JwtSecret`` objects when related Applicatie objects are bulk deleted in the Admin
+
+**Project maintenance**
+
+* [#1761] Add ``resultaattype.resultaattypeomschrijving`` into ``generate_data`` command
```
Comment (Coperh) [inline CHANGELOG.rst:23]: I think this should refer to issue #1798 from PR #1803

Comment (Coperh) [reply CHANGELOG.rst:23]: Yes #1761 is already added in 1.15 below

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
 
-1.16.0 (WIP)
-------------
+1.16.0 (2024-11-25)
+-------------------
 
 **New features**
 
+* [#1733] Add experimental support for registering authentication context with mandates (machtiging)
+   and searching through them
```
Comment (Coperh) [inline CHANGELOG.rst:10]:
```suggestion
* [#1733] Add experimental support for registering authentication context with mandates (machtiging)
  and searching through them
```
I think the indentation is bad making the first point bold

**Review by joeribekker — APPROVED** [review]

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,13 +1,32 @@
...
+* [#1783] Add ``zaaktype__not_in`` filter to ``/zaken/api/v1/zaken/_zoek`` endpoint
+* [#1774] Add ``trefwoorden__overlap`` filter to ``/documenten/api/v1/enkelvoudiginformatieobjecten`` endpoint
+* [#1776] Support searching zaken on ``zaaktype.omschrijving`` in the Admin
+* [#1611] Add importing authorization data via yaml files as a configuration step
 * [#1817] Updated OAF version to 0.9.0. This upgrade allows admin users managing their sessions through the admin.
```
Comment (Coperh) [inline CHANGELOG.rst:15]: I think maybe we should separate the library update into Project maintenance. 

Although I did approve the other PR :/

**Review by Coperh — CHANGES_REQUESTED** [review]
Nothing missing.

One change is linked to the wrong pr and there is a format error.

File: CHANGELOG.rst (hunk 1 above)
Comment (annashamray) [reply CHANGELOG.rst]: Changed. Thank for catching it

**Review by Coperh — APPROVED** [review]

**Pull request merged**